LanguageFeatures/Patterns/constant_A06_t01.dart

```diff
@@ -61,6 +61,18 @@ main() {
     case C():
 //       ^^^
 // [analyzer] unspecified
+// [cfe] unspecified
+    case 1 + 2:
+//       ^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+    case 1 & 2:
+//       ^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+    case 1 | 2:
```

Comment:
Given that `1` and `2` are <constantPattern>s, I'd say that this is a <logicalOrPattern>, which should not be an error.

We have many other forms: All constant expressions that cannot be derived by the new `constantPattern` rule.

Reply:
> Given that 1 and 2 are s, I'd say that this is a , which should not be an error.

Do you mean that this is a "logical-or" pattern which is not an error?

Reply:
Yes, I just edited the text to have the missing backquotes. ;)